app/src/main/java/com/babylon/wallet/android/presentation/transaction/submit/TransactionSubmitDelegate.kt

```diff
@@ -141,39 +141,35 @@ class TransactionSubmitDelegate @Inject constructor(
                 feePayerAddress = feePayerAddress
             ),
             deviceBiometricAuthenticationProvider = deviceBiometricAuthenticationProvider
-        ).then { notarisation ->
-            submitTransactionUseCase(
-                txIDHash = notarisation.txIdHash,
-                notarizedTransactionHex = notarisation.notarizedTransactionIntentHex,
-                endEpoch = notarisation.endEpoch
-            )
-        }.onSuccess { submitTransactionResult ->
+        ).then { notarizationResult ->
+            submitTransactionUseCase(notarizationResult = notarizationResult)
+        }.onSuccess { notarisation ->
             _state.update {
                 it.copy(
                     isSubmitting = false,
-                    endEpoch = submitTransactionResult.endEpoch
+                    endEpoch = notarisation.endEpoch
```

Comment:
we use `notarize` and `notarise`. Are both correct? It seem that this could be `notarization`